app/views/quotes/index.html.erb

```diff
@@ -9,7 +9,7 @@
       </footer>
     </blockquote>
     <div class="created">
-      <%= time_ago_in_words(quote.created_at) %> ago
+       <%= time_ago_in_words(quote.created_at) unless quote.created_at.blank?%> ago
```

Comment:
This looks like a good workaround, we'll ship this to production right now to prevent the error from occurring -- but can you make the DB changes that you suggested, and add some tests to insure that this cannot happen again?